cmd/umserver/main.go

```diff
@@ -4,6 +4,88 @@
 // All users data will be stored in a database.
 package main
 
+import (
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+)
+
+const GracefulShutdownTimeOut = 10 * time.Second
```

Comment:
`GracefulShutdownTimeOut` constant should be unexported

Reply:
oh, Thank you for advice. I will fix it.